src/incubator/hooks/useHiddenLocation.ts

```diff
@@ -10,13 +10,26 @@ export interface HiddenLocation extends HiddenLocationRecord {
   wasMeasured: boolean;
 }
 
+// Adding this for headless tests that are not triggering onLayout
+// Unfortunately Dialog._inTest only exist after Dialog was created,
+// so this cannot be done statically
+function getWasMeasuredDefaultValue() {
+  let wasMeasuredDefaultValue = false;
+  if (__DEV__) {
+    wasMeasuredDefaultValue = require('../Dialog').default._inTest ?? false;
```

Comment:
You are requiring a component that use this hook (doesn't it cause cyclic require issue?)
And you also reference Dialog inside a general hook, since useHiddenLocation is not a dedicated hook of Dialog. It's a bit weird don't you think?

Reply:
Technically it is a cycle, yes, and I thought it's not optimal as well.
Excuses: this is the solution we've talked about and this is only in dev.

Another option I thought about is sending the defaultValue via a prop with a default value, but I'm pretty sure this means the boolean value cannot be set on Dialog itself.